antora-ui-camel/src/css/header.css

```
@@ -417,7 +417,7 @@ body {
 #search-cancel {
   position: relative;
   bottom: 1.4rem;
-  left: 10.6rem;
+  left: 7.5rem;
   height: 1.1rem;
```

Perhaps instead of relying on specific numbers we can rely on the principles, this way if we change the search input field size the clear icon will be positioned correctly.

```suggestion
  bottom: calc(50% - 0.15rem);
  left: calc(100% - 1.25rem);
  height: 1rem;
```

Not sure if 0.15 and 1.15 could be derived from a variable, that would be better.